llvm/include/llvm/Transforms/Utils/LoopUtils.h

```diff
@@ -394,6 +395,8 @@ Value *getShuffleReduction(IRBuilderBase &Builder, Value *Src, unsigned Op,
 /// Fast-math-flags are propagated using the IRBuilder's setting.
 Value *createSimpleTargetReduction(IRBuilderBase &B, Value *Src,
                                    RecurKind RdxKind);
+Value *createSimpleTargetReduction(VectorBuilder &VB, Value *Src,
```

Comment:
nit: might be good to highlight that this version is used to create vector-predication intrinsics?

Reply:
518051b849bdf0148ff1ec3b499da667873eec0a